src/main/scala/io/movies/api/MovieServiceHttp4s.scala

```diff
@@ -0,0 +1,52 @@
+package io.movies.api
+
+import io.movies.model._
+import io.movies.services.MovieService
+import cats.effect._
+import cats.syntax.all._
+import org.http4s._
+import org.http4s.dsl.Http4sDsl
+import org.http4s.server.blaze._
+import org.http4s.circe._
+import io.circe.generic.auto._
+
+class MovieServiceHttp4s[F[_]: Sync: MovieService]
+    extends Http4sDsl[F] {
+  val ms: MovieService[F] = implicitly[MovieService[F]]
+  implicit val decoder: EntityDecoder[F, Movie] = jsonOf[F, Movie]
+  implicit val encoder: EntityEncoder[F, RegisteredMovie] = jsonEncoderOf[F, RegisteredMovie]
+  implicit val encoderList: EntityEncoder[F, List[RegisteredMovie]] = jsonEncoderOf[F, List[RegisteredMovie]]
+
+  lazy val MOVIE_SERVICE = "MovieService"
+  lazy val root = Root / MOVIE_SERVICE
```

Comment:
After the changes, you're not using `root` in the routes

Reply:
See https://github.com/gutiory/MoviesServer/pull/1/commits/a28637fa15b5b3c4cc9c80257e93a24c04abb8ad